Header header
sensor_msgs/Image img
int32 lm_count
std_msgs/UInt64MultiArray lm_id_data
geometry_msgs/Vector3[] lm_2d_data
geometry_msgs/Vector3[] lm_3d_data
geometry_msgs/Transform T_c_w

